src/services/map-events-mananger/map-events-manager.ts

```diff
@@ -10,8 +10,11 @@ import { CesiumEventModifier } from './consts/cesium-event-modifier.enum';
 import { UtilsService } from '../../utils/utils.service';
 import { PlonterService } from '../plonter/plonter.service';
 
+let findIndex = require('lodash.findindex');
```

Comment:
if you must use findIndex then add `lodash.findindex` to package.json and please use import instead of require